fabric/src/main/java/net/minescript/common/Minescript.java

```diff
@@ -3981,6 +4010,116 @@ private static Optional<String> handleScriptFunction(
         }
         return Optional.of(toJsonString(getScreenName().orElse(null)));
 
+      case "container_get_item": // Item in slot in container
```

Comment:
`container_get_item(slot)` seems redundant with `container_get_items()` which a Python script can use to easily filter for specific items. Similarly, there's no script function for getting an individual item from the player's inventory, only all items via `player_inventory()`.